### PR TITLE
Add documentation about the IATI tooling and organisation space

### PR DESCRIPTION
**Pull request by nathanbegbie**
Addresses #33

### CODE REVIEW

File: docs/overview.md [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+The hope is that by the end of reading this, you will have a much better idea of
+* what the tool 'space' looks like for IATI
+* where the gaps might exist for additional IATI tools
+* what tools we can re-use in attempting to achieve our goals
+* clarity on data sources, where it sits and which tools rely on which data sources
+* who we could/should partner with from a technical and organisational perspective
+
+# So what is IATI?
+At a very broad level, IATI is a standard.
+A standard is an agreed upon way of communicating [^standardsdocument].
```
Comment (kevincarolan) [inline docs/overview.md]: Is "[^standardsdocument]" a typo?

File: docs/overview.md [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+This means that there would be a common way of communicating something like "we worked on combatting HIV AIDS in South Africa,
+spending $1 million in expenses and disbursing another $1 million to X and Y organisation".
+
+It was agreed that this data would be published as XML, but that it would also adhere to a Schema.
+
+## Identifiers
+Identifiers are a string of letters and numbers that uniquely identify something.
+They are used for both activities and organisations.
+They allow us to identify and connect disparate bits of information across documents.
+Note that in some cases, widentifiers might not be included in the
```
Comment (kevincarolan) [inline docs/overview.md]: the sentence ends without a clause, included in the what?
```suggestion
Note that in some cases, identifiers might not be included in the  
```

File: docs/overview.md [hunk 3]
```diff
@@ -0,0 +1,259 @@
...
+spending $1 million in expenses and disbursing another $1 million to X and Y organisation".
+
+It was agreed that this data would be published as XML, but that it would also adhere to a Schema.
+
+## Identifiers
+Identifiers are a string of letters and numbers that uniquely identify something.
+They are used for both activities and organisations.
+They allow us to identify and connect disparate bits of information across documents.
+Note that in some cases, widentifiers might not be included in the  
+There are norms that govern identifiers. The way to think about is
```
Comment (kevincarolan) [inline docs/overview.md]: What is the way to think about it?

File: docs/overview.md [hunk 4]
```diff
@@ -0,0 +1,259 @@
...
+This may not always be the case, as some organisations will use only the organisations name and possibly
```
Comment (kevincarolan) [inline docs/overview.md]: and possibly what? (it appears like text got dropped or is missing in a few areas)

File: docs/overview.md [hunk 5]
```diff
@@ -0,0 +1,259 @@
...
+|    Lithuania, Ministry of Foreign   Affairs                                             |    1077                      |
+|    International Climate Initiative /   BMU - IKI Secretariat                           |    618                       |
+
+However, dealing with versions is difficult, and may involve additional work.
+It may be pragmatic to ignore the Version 1 data in some cases.
+
+## Validation
+When we ask the question 'is this information valid?', there are a number of levels to consider;
+XML - does the document itself valid XML? There are plenty of tools that can do this.
+TATI Schema - there are certain required elements and structure in the document, as well as how they should be added to the document.
```
Comment (kevincarolan) [inline docs/overview.md]:
```suggestion
IATI Schema - there are certain required elements and structure in the document, as well as how they should be added to the document.
```

File: docs/overview.md [hunk 6]
```diff
@@ -0,0 +1,259 @@
...
+This data is pulled from `https://iatiregistry.org/api/3/`, which is indicated in the code [here](https://github.com/IATI/IATI-Registry-Refresher/blob/master/grab_urls.php#L33).
+
+### [IATI Public Validator](https://validator.iatistandard.org)
+Provides some kind of structural validation. Doesn't do semantic validation.
+
+## [Development Initaitives](www.devinit.org)
+> an independent international development organisation that focuses on the role of data in driving poverty eradication and sustainable development. [^devinitdescriptionsource]
+
+There are a number of individuals at Development Initaitives (DI) who are on the IATI secretariat.
+This means that
```
Comment (kevincarolan) [inline docs/overview.md]: it appears that text is missing here

File: docs/overview.md [hunk 7]
```diff
@@ -0,0 +1,259 @@
...
+## [Publish What You Fund](http://www.publishwhatyoufund.org)
+
+### [data-quality-tester](http://dataqualitytester.publishwhatyoufund.org/)
+> Test your IATI data against the Publish What You Fund Aid Transparency Index tests. [^ dqtdescription]
+
+### [IATI Canary](https://iati-canary.herokuapp.com/)
+> checks to ensure data is both available and compliant with the IATI schema. 
+If there’s a problem, you’ll be notified by email [^ canarysource]
+
+Note that thhis tool is opt-in.
```
Comment (kevincarolan) [inline docs/overview.md]:
```suggestion
Note that this tool is opt-in.
```

File: docs/overview.md (hunk 1 above)
Comment (nathanbegbie) [reply docs/overview.md]: Sorry - it's a way of creating footnotes in Rmd, not md. I think for now, I'll put the footnotesin parenthesis.

**Comment by nathanbegbie** [comment]
@kevincarolan have made the changes you suggested, fixed typos and removed dangling sentences.

**Pull request merged**